lib/template-generator.js

```diff
@@ -12,8 +12,7 @@ module.exports = function templateGenerator (opts, cb) {
   var dateFormat = opts.dateFormat
   var templateFile = opts.templateFile || path.join(__dirname, 'template.js')
   var migrationsDirectory = opts.migrationsDirectory || 'migrations'
-  var extension = opts.extension || '.js'
```

Comment:
Since you removed this, it looks like you need to change line 27:

```
var p = path.join(process.cwd(), migrationsDirectory, slug(formattedDate + (name ? '-' + name : '')) + extension)
```


```
var p = path.join(process.cwd(), migrationsDirectory, slug(formattedDate + (name ? '-' + name : '')) + opts.extension)
```

FWIW, I would recommend running `npm t` before pushing your code, because it should catch this at the linter stage.